### ISSUE TITLE
Find friends who are training models and share ideas with them.Welcome 

### ISSUE THREAD

**Issue by LiuJiaqi0505** [hitlecapital/wav2lip_sam#133]
I have created a communication group on wechat, which you can enter directly through wechat. Friends from all over the world are welcome to join the discussion
![image](https://github.com/hitlecapital/wav2lip_sam/assets/38346279/c0a75898-962d-463f-9fc9-0387454e6ec1)

**Comment by PolarRobin** [comment]
Your code is expired, is the group active?

**Comment by LiuJiaqi0505** [comment]
> Your code is expired, is the group active?

yes, welcome to our group. you could add my wechat account: liuqipao55

**Comment by see2run** [comment]
> > Your code is expired, is the group active?
> 
> yes, welcome to our group. you could add my wechat account: liuqipao55

Hey, can you help me? Percep, Fake, and Real are always 0.0 during the Wav2Lip training. Please
